dmoj/cptbox/isolate.py

```diff
@@ -2,22 +2,21 @@
 import os
 import sys
 from enum import Enum
-from operator import attrgetter
-from typing import Optional, Tuple
+from typing import Any, Callable, Mapping
 
 from dmoj.cptbox._cptbox import AT_FDCWD, Debugger, bsd_get_proc_cwd, bsd_get_proc_fdno
 from dmoj.cptbox.filesystem_policies import FilesystemPolicy
 from dmoj.cptbox.handlers import (
     ACCESS_EACCES,
     ACCESS_EFAULT,
-    ACCESS_EINVAL,
     ACCESS_ENAMETOOLONG,
     ACCESS_ENOENT,
     ACCESS_EPERM,
     ALLOW,
     ErrnoHandlerCallback,
 )
 from dmoj.cptbox.syscalls import *
+from dmoj.cptbox.syscalls import by_id
```

Comment:
This is already implied by * above.

Reply:
No it's not: `syscalls` defines `__all__`, which is what is actually imported.